javascript/utils/queued-move.js

```diff
@@ -71,7 +72,6 @@ let MoveArgIdxs = {
   SILVER_SENT: 6,
 }
 let contractPrecision = 1000;
```

Comment:
@Bind mind pulling this from `import { CONTRACT_PRECISION }  from 'https://cdn.skypack.dev/@darkforest_eth/constants';`

The more we move to official packages, the less effort this should take next time 😛